tests/src/main/java/net/neoforged/neoforge/debug/capabilities/ItemInventoryTests.java

```diff
@@ -1,94 +0,0 @@
-/*
- * Copyright (c) NeoForged and contributors
- * SPDX-License-Identifier: LGPL-2.1-only
- */
-
-package net.neoforged.neoforge.debug.capabilities;
-
-import net.minecraft.core.NonNullList;
-import net.minecraft.core.component.DataComponents;
-import net.minecraft.world.item.Item;
-import net.minecraft.world.item.ItemStack;
-import net.minecraft.world.item.Items;
-import net.minecraft.world.item.component.ItemContainerContents;
-import net.neoforged.neoforge.capabilities.Capabilities.ItemHandler;
-import net.neoforged.neoforge.capabilities.RegisterCapabilitiesEvent;
-import net.neoforged.neoforge.items.ComponentItemHandler;
-import net.neoforged.neoforge.items.IItemHandler;
-import net.neoforged.neoforge.registries.DeferredItem;
-import net.neoforged.testframework.DynamicTest;
-import net.neoforged.testframework.TestFramework;
-import net.neoforged.testframework.annotation.ForEachTest;
-import net.neoforged.testframework.annotation.OnInit;
-import net.neoforged.testframework.annotation.TestHolder;
-import net.neoforged.testframework.gametest.EmptyTemplate;
-import net.neoforged.testframework.gametest.GameTest;
-import net.neoforged.testframework.registration.DeferredItems;
-import net.neoforged.testframework.registration.RegistrationHelper;
-
-@ForEachTest(groups = "capabilities.iteminventory")
-public class ItemInventoryTests {
```

Comment:
Don't remove these old tests. Keep them, and just wrap the new handler as an old `IItemHandler`.